src/pages/playground/index.tsx

```diff
@@ -23,9 +25,33 @@ import { trpc } from 'lib/client/trpc';
 import type PageWithLayout from 'types/persistentLayout';
 
 const Playground: PageWithLayout = ({}) => {
+  const router = useRouter();
+
+  const isCategories = (
```

Comment:
The `isCategories` guard should be moved outside the component, primarily for readability.
Maybe `isValidPlaygroundRequestCategoryValue` would be a more speaking name?